src/relay/backend/task_extraction.cc

```diff
@@ -42,6 +43,14 @@ Array<meta_schedule::ExtractedTask> ExtractTask(
   // is_vm=true for backward compatibility
   Array<Pass> pass_seqs = relay::backend::GetPassPrefix(/*is_homogenous=*/true, /*is_vm=*/true);
   pass_seqs.push_back(transform::FuseOps());
+
+  if (auto link_params_obj = target->attrs.Get("link-params")) {
+    ICHECK(link_params_obj.as<IntImmNode>());
+    int link_params = link_params_obj.as<IntImmNode>()->value;
+    auto ctx = transform::PassContext::Current();
+    ctx->config.Set("relay.FuseOps.link_params", IntImm(DataType::Int(32), link_params));
+  }
```

Comment:
The reason for this change is explained in https://github.com/apache/tvm/pull/12416#issuecomment-1219282120.

It depends on soon-to-be deprecated target option. So this change could be controversial.